packer/linux/conf/bin/bk-install-elastic-stack.sh

```diff
@@ -100,6 +100,11 @@ if [[ "${BUILDKITE_AGENT_ENABLE_GIT_MIRRORS_EXPERIMENT}" == "true" ]] ; then
   fi
 fi
 
+# If the agent token path is set, use that instead of BUILDKITE_AGENT_TOKEN
+if [[ -z "${BUILDKITE_AGENT_TOKEN_PATH}" ]] ; then
```

Comment:
Do you mean `-n` here? It's only supposed to do this if there is a path, right?

Reply:
Oops yep - probably a bad copy-pasta